### ISSUE TITLE
Write SBOL document to standard out

### ISSUE THREAD

**Issue by tramyn** [SynBioDex/pySBOL3#115]
I am looking for a function that will allow me to write the contents in an sbol document to a standard output. The library currently supports writing an SBOL document to a file but this function is not desirable when I want to display an sbol document content to a web browser. A temporary solution provided by @tcmitchell is to write an sbol document to a local file then read it back to a desired format. Although this is a solution that I can work off of, it is not a direct solution and would like to request to have this feature supported in the library.

**Comment by jakebeal** [comment]
If you can write to a file, shouldn't that be able to write standard output?

**Comment by tramyn** [comment]
From what I was told by @tcmitchell, this is not the case. @tcmitchell could you confirm?

**Comment by tcmitchell** [comment]
`Document.write()` writes to a file. `Document.write_string()`, which would return a string, does not exist. I'm not sure if I've answered the question @jakebeal asked because the question is vague. 

I provided @tramyn with a short-term workaround, which is to write to file, then read the file as a string and send that string to stdout. I plan to implement `Document.write_string()`, or something similar, to fulfill this issue.
